Backend/utils/MentiiAuth.py

```diff
@@ -0,0 +1,94 @@
+from flask import g
+from itsdangerous import (TimedJSONWebSignatureSerializer as Serializer, BadSignature, SignatureExpired)
+from boto3.dynamodb.conditions import Key, Attr
+import hashlib
+import ConfigParser as cp
+from mentii import user_ctrl
+
+class MentiiAuthentication:
```

Comment:
Uhh I'll look into it. I just made it a class initially to make it cleaner. Just wondering, are there any pros/cons to having a class with static methods vs just a module?